toonz/sources/toonz/cellselection.cpp

```diff
@@ -430,6 +430,7 @@ class InsertUndo final : public TUndo {
 
 class RenumberUndo final : public TUndo {
   std::map<TXshCell, TXshCell> m_undoTable, m_redoTable;
+  std::vector<TXshChildLevel *> *m_childLevels;
```

Comment:
It's not your fault, but is blamed for bad naming of class (already done in Toonz Harlequin's source, not me!).
Sorry to say, I realized that there are two `RenumberUndo` for now.

I imagine you meant to fix `RenumberUndo` in `filmstripcommand.cpp` . 
This `RenumberUndo` in `cellselection.cpp` is used for `Autorenumber` command and is not need to be changed -
 except the confusing class name.
Regarding the `RenumberUndo` in `filmstripcommand.cpp`, it seems that you don't need to add any additional modification as it calls `updateXSheet()` .

Reply:
@shun-iwasawa The renumber issue also occurs on the autorenumber command from the xsheet.  I was aware that I was updating the the command for that as well.  Without this, changes made to the level numbering don't always propagate to the same level in subxsheets.

Reply:
Sorry for not taking note of it. I'll review this again.